src/index.ts

```diff
@@ -1,7 +1,95 @@
+import { DataXY } from 'cheminfo-types';
+import sum from 'ml-array-sum';
+import variance from 'ml-array-variance';
+import covariance from 'ml-array-xy-covariance';
+import { EigenvalueDecomposition } from 'ml-matrix';
+
 /**
- * My module
- * @returns A very important number
+ * Calculates the Directional Ellipse for a set of points with a specific standard deviation
+ * @param {DataXY} [points]
```

Comment:
In typedoc we should specify the type. It should simply be:

`@param [points] - here the description`

We have an action that generates the doc using `typedoc`. You can test locally using

`npm i --global typedoc``

`typedoc src`

This will create a folder docs that you can preview for example with the vscode plugin 'live server'.